### PR TITLE
Feat/ducks

### PR DESCRIPTION
**Pull request by anneback**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -78,13 +78,7 @@ const itemsReducer = (state = {}, action) => {
...
+export const dataReducer = combineReducers({
   itemsIds: itemsIdsReducer,
   items: itemsReducer
 });
-
-// ROOT
-export const rootReducer = combineReducers({
-  ui: uiReducer,
-  data: dataReducer
-});
```
Comment (iamstarkov) [inline src/reducers/index.js:90]: you dont need this file, pls remove it

Comment (iamstarkov) [reply src/reducers/index.js:90]: as well as src/actions/index.js file

**Review by iamstarkov — APPROVED** [review]
apart from some dead files, it is very good

**Comment by iamstarkov** [comment]
LGTM

File: src/reducers/index.js (hunk 1 above)
Comment (anneback) [reply src/reducers/index.js:90]: Done

**Pull request merged**